pandas/tests/test_algos.py

```diff
@@ -47,6 +47,10 @@
 )
 import pandas.core.common as com
 
+pytestmark = pytest.mark.filterwarnings(
+    "ignore:Series.view is deprecated and will be removed in a future version.:FutureWarning"  # noqa: E501
```

Comment:
This seems a bit a "dangerous" ignore, because we want to be sure the algos itself aren't raising the warning, and algos seems a part of the codebase that is using `view()` regularly (although already not with Series objects apparently, assuming it is correct no change is needed there)

Reply:
I can ignore them specifically, will update later